docs/output.md

```diff
@@ -16,12 +17,14 @@
 * [Output Files](#output-files)
 
 ## Introduction
+
 The output of EAGER2 consists of two main components: output files (e.g. BAM or FASTQ files), and summary statistics of the whole run presented in a [`MultiQC`](https://multiqc.info) report. (Some) intermediate files and module-specific statstics files are also retained.
 
 ## Directory Structure
+
 The directory structure of EAGER2 is as follows
 
-```
+```bash
 <RUN_OUTPUT_DIRECTORY>/
 ├── MultiQC/
 ├── <MODULE_1>/
```

Comment:
To specify they would come from distinct outputs, rather than the output from one tool being spread out over multiple ones. But I guess this is not that important. Say if you prefer to have it without hte numbers.